electron/__snapshots__/syntheticsGenerator.test.ts.snap

```diff
@@ -0,0 +1,110 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Synthetics JavaScript formatter accepts custom step organization 1`] = `
+"const { journey, step, expect } = require('@elastic/synthetics');
+
+journey('Recorded journey', async ({ page, context }) => {
+  let page1;
+  step('Open new page', async () => {
+    await page.goto('https://vigneshh.in/');
+  });
+  step('Assert text=Babel Minify isVisible', async () => {
+    expect(await page.isVisible('text=Babel Minify')).toBeTruthy();
+    expect(await page.isEditable('text=Babel Minify')).toBeTruthy();
+  });
+  step('Assert text=Babel Minify textContent', async () => {
+    expect(await page.textContent('text=Babel Minify')).toMatch('Babel');
+    [page1] = await Promise.all([
+      page.waitForEvent('popup'),
+      page.click('text=Babel Minify')
+    ]);
+    await Promise.all([
+      page1.waitForNavigation(/*{ url: 'https://github.com/babel/minify' }*/),
+      page1.click('a:has-text(\\"smoke\\")')
+    ]);
+  });
+  step('Close page', async () => {
+    await page1.close();
+  });
+});"
+`;
+
+exports[`Synthetics JavaScript formatter does not hoist when all accesses are in one step 1`] = `
+"let page1;
+let page2;
+step('Go to https://vigneshh.in/', async () => {
+  await page.goto('https://vigneshh.in/');
+  [page1] = await Promise.all([
+    page.waitForEvent('popup'),
+    page.click('text=Tailor')
+  ]);
+  await page1.click('text=Packages 0');
+  expect(page1.url()).toBe('https://github.com/orgs/zalando/packages?repo_name=tailor');
+  await page1.close();
+  [page2] = await Promise.all([
+    page.waitForEvent('popup'),
+    page.click('text=Babel Minify')
+  ]);
```

Comment:
It is quite clear that page1 doesn't exist anymore so it is obvious to use `page` in line 45 and 46. But out of curiosity, what happens when page1 isn't closed yet - then do we know which one from `page` or `page1` need to 
 be used in lines 45 and 46 ?

Reply:
Pages are the context in PW, It does not matter if the page 1 window is kept open or closed, If the
- new window is opened from page1, then page2 would be waiting for popup event from `page1`. 
- If the window is opened from first page `page`, then page2 would be waiting for popup from `page` . 

Hope this clears up.